# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2018, Optuna Contributors.
# This file is distributed under the same license as the Optuna package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Optuna 2.4.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-01-14 18:04-0500\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.8.0\n"

#: ../../source/reference/generated/optuna.TrialPruned.rst:2
msgid "optuna.TrialPruned"
msgstr ""

#: of optuna.TrialPruned:1
msgid "Exception for pruned trials."
msgstr "被剪枝的 trial 异常."

#: of optuna.TrialPruned:3
msgid ""
"This error tells a trainer that the current :class:`~optuna.trial.Trial` "
"was pruned. It is supposed to be raised after "
":func:`optuna.trial.Trial.should_prune` as shown in the following "
"example."
msgstr "该错误告诉用户 当前 :class:`~optuna.trial.Trial` 已经被剪枝. 就像下面的例子里展示的那样, 它被设定在"
" :func:`optuna.trial.Trial.should_prune` 之后被触发."

#: of optuna.TrialPruned:7
msgid ""
":class:`optuna.TrialPruned` is an alias of "
":class:`optuna.exceptions.TrialPruned`."
msgstr ":func:`optuna.TrialPruned` 是 :func:`optuna.exceptions.TrialPruned` 的别名."

#: of optuna.TrialPruned:10
msgid "示例"
msgstr ""

